process-engine/process_engine_core
# ♻️ Default empty external ServiceTask results to `{}`

## Changes

1. When an ExternalTask finishes with `{}`, `null` or `undefined`, the resulting `onExit` token of the corresponding ServiceTask will be `{}`. 
2. Fix reference error, when a message received from an ExternalTaskWorker is empty.

## Issues

Part of https://github.com/process-engine/process_engine_runtime/issues/502

PR: #317

## How to test the changes

- Run an external ServiceTask
- Finish the ExternalTask with `{}`, `null` or `undefined`
- Notice that the ServiceTask`s `onExit` token will always be `{}`
